.github/workflows/salt-sls-files-checks.yml

```diff
@@ -0,0 +1,21 @@
+name: Check the content of Salt SLS files
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  salt_sls_files_checks:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
```

Comment:
There is newer version (`v2`) of `actions/checkout`, it uses `fetch-depth: 1` per default (so it can be omitted here) and apparently contains some other improvements. See again the [rubocop workflow](https://github.com/uyuni-project/uyuni/blob/master/.github/workflows/rubocop.yml) for an example.